bin/cli.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+const program = require('commander')
+const app = require('../lib/mine.js')
+const assert = require('assert')
+const pckg = require('../package.json')
+
+program
+  .version(pckg.version)
+  .usage('[options]')
+  .option('-m, --mine-address <address>', 'Blockchain address for the mine to use')
+  .option('-c, --contract-address <address>', 'Sonar smart contract address for the mine to use')
+  .option('-i, --ipfs-url [url]', 'Url of the IPFS node (Default: "{host: \'localhost\', port: \'5001\', protocol: \'http\'})")')
```

Comment:
Do we really have to input a string encoded JSON here or shouldn't it be to pass `http://localhost:5001` as you do with `--ethereum-url`?